README.md

```diff
@@ -182,6 +182,32 @@ require("lualine").setup({
 })
 ```
 
+</details>
+	
+### [galaxyline](https://github.com/glepnir/galaxyline.nvim)
+
+<details>
+<summary>An example galaxyline setup </summary>
+
+```lua
+local navic = require("nvim-navic")
+local gl = require("galaxyline")
+local condition = require("galaxyline.condition")
```

Comment:
I don't think this is required. Also remove the extra empty line.

Reply:
line 202 will not work without this variable

Reply:
Nope, there `condition` is a key in the table `nvimNavic`. It does not rely on the import statement. The imported condition is for using built-in conditions like condition.buffer_not_empty etc.

Reply:
I see. Removed the variable now.